src/aws/batchDeleteHandler.ts

```diff
@@ -16,7 +16,9 @@ import { setTimeout } from 'timers/promises';
 export type BatchDeleteMessage = {
   traceId: string;
   userId: number;
-  itemIds: number[];
+  itemIds?: number[];
```

Comment:
made this optional so can be be re-used by emitAccountDeleteEvent, can also create a new type if it helps to have itemIds as non-optional field

Reply:
I think this has to be a required field for BatchDeleteMessage. If you want to reuse it you could make an `Omit` type, or make a base type `UserMessageBase` or something that `BatchDeleteMessage` extends?

Reply:
done